### ISSUE TITLE
Can't activate portrait mode layout

### ISSUE THREAD

**Issue by axg20202** [Jay-Tech/ioSenderTouch#6]
This is excellent. Good for non touch screens too. There doesn't appear to be a settting for portrait mode. How is this acitivated? The layout doesn't appear to reorganise automatically but can't see a setting for it. Is this a bug?

**Comment by axg20202** [comment]
Ok, I discovered the issue. Windows won't display the UI properly when the desktop is set to 'Portrait flipped', but 'Portrait' works. Is this something that can be resolved in a new release?

**Comment by Jay-Tech** [comment]
Yeah, did not account for flipped  screen orientation .  Just as a note something windows does that I have not worked around is. that it will give the orientation of the primary monitor.  So if you are hooked up to a laptop and its the primary screen and the external monitor is in portrait the app will not chnage layouts until the external monitor is  set as primary. 
Try this to see if it fixes your issue
https://github.com/Jay-Tech/ioSenderTouch/releases/tag/V1.0.4

**Comment by axg20202** [comment]
That's great. Thanks. WIll try it out.
